cips/cip-29.md

```diff
@@ -0,0 +1,245 @@
+|  **10** |               2.79 |                2.67905 | Regular annual disinflation applied (6.7%).                             |
+|  **11** |               2.51 |                2.49956 | Regular annual disinflation applied (6.7%).                             |
+|  **12** |               2.26 |                2.33209 | Regular annual disinflation applied (6.7%).                             |
+|  **13** |               2.03 |                2.17584 | Regular annual disinflation applied (6.7%).                             |
+|  **14** |               1.83 |                2.03005 | Regular annual disinflation applied (6.7%).                             |
+|  **15** |               1.65 |                1.89404 | Regular annual disinflation applied (6.7%).                             |
+|  **16** |               1.50 |                1.76714 | Regular annual disinflation applied (6.7%).                             |
+|  **17** |               1.50 |                1.64874 | Regular annual disinflation applied (6.7%).                             |
+|  **18** |               1.50 |                1.53828 | Regular annual disinflation applied (6.7%).                             |
+
+To understand the numbers on how this would affect stakers please refer to the tables below:
+
+Current schedule:
+
+| Year   | Inflation Rate | APR (35% Staked) | APR (50% Staked) | APR (65% Staked) |
+|:-------|:---------------|:-----------------|:-----------------|:-----------------|
+| **0**  | 8.0            | 22.86            | 16.0             | 12.31            |
+| **1**  | 7.2            | 20.57            | 14.4             | 11.08            |
+| **2**  | 6.48           | 18.51            | 12.96            | 9.97             |
+| **3**  | 5.83           | 16.66            | 11.66            | 8.97             |
+| **4**  | 5.25           | 15.0             | 10.5             | 8.08             |
+| **5**  | 4.72           | 13.49            | 9.44             | 7.26             |
+| **6**  | 4.25           | 12.14            | 8.5              | 6.54             |
+| **7**  | 3.83           | 10.94            | 7.66             | 5.89             |
+| **8**  | 3.44           | 9.83             | 6.88             | 5.29             |
+| **9**  | 3.1            | 8.86             | 6.2              | 4.77             |
+| **10** | 2.79           | 7.97             | 5.58             | 4.29             |
+| **11** | 2.51           | 7.17             | 5.02             | 3.86             |
+| **12** | 2.26           | 6.46             | 4.52             | 3.48             |
+| **13** | 2.03           | 5.8              | 4.06             | 3.12             |
+| **14** | 1.83           | 5.23             | 3.66             | 2.82             |
+| **15** | 1.65           | 4.71             | 3.3              | 2.54             |
+| **16** | 1.5            | 4.29             | 3.0              | 2.31             |
+
+Updated schedule:
```

Comment:
There are two updated schedules below. The second table has higher precision so let's drop the first one.